source/components/harnesses.md

```diff
@@ -1,23 +1,35 @@
 # Harnesses
 
-Harnesses allow you to run relevant groups of tests all together. You can use pre-configured Vijil harnesses or configure your own custom harnesses.
+Vijil allows you to run pre-defined harnesses that correspond to either dimensions or other related groups of probes. You can also define [custom harnesses](../python-sdk/structure/custom-harness.md) to better fit your organization's priorities.
 
-Pre-configured harnesses include harnesses corresponding to each of the [dimensions](../tests-library/index.md), and other harnesses that are subsets of those dimensions.
+## Pre-defined harnesses
 
-## Dimension harnesses
+Harnesses allow you to run relevant groups of probes all together. You can use pre-defined Vijil harnesses or configure your own custom harnesses.
 
-- `security`
-- `privacy`
-- `hallucination`
-- `robustness`
-- `toxicity`
-- `stereotype`
-- `fairness`
-- `ethics`
+Every [dimension](../tests-library/index.md) is a pre-configured harness.  In addition, each scenario is also a harness. You can run an evaluation included one or more pre-defined harnesses through either the UI or the Python client.
+
+### Dimension harnesses
```

Comment:
Trying this organization of content to make it more obvious that we have dimension harnesses and then other types of harnesses